client/package.json

```diff
@@ -47,5 +47,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:5000"
+  "proxy": "http://localhost:5000",
+  "devDependencies": {
+    "typescript": "^4.2.3"
+  }
```

Comment:
Why have you installed typescript on the client-side?

Reply:
For eslint to work through command line, it required typescript to be installed. I installed it as a dev dependency so that it will not be a problem for production.

Here's the error message without typescript installed - 
```
Oops! Something went wrong! :(

ESLint: 7.23.0

Error: Failed to load parser '@typescript-eslint/parser' declared in 'client\package.json » eslint-config-react-app#overrides[0]': Cannot find module 'typescript'
Require stack:
- C:\Users\avina\Projects\Stackoverflow-Clone\client\node_modules\@typescript-eslint\typescript-estree\dist\parser.js
- C:\Users\avina\Projects\Stackoverflow-Clone\client\node_modules\@typescript-eslint\parser\dist\parser.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\@eslint\eslintrc\lib\config-array-factory.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\@eslint\eslintrc\lib\index.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\eslint\lib\cli-engine\cli-engine.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\eslint\lib\eslint\eslint.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\eslint\lib\eslint\index.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\eslint\lib\cli.js
- C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\eslint\bin\eslint.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:965:15)
    at Function.Module._load (internal/modules/cjs/loader.js:841:27)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\v8-compile-cache\v8-compile-cache.js:159:20)
    at Object.<anonymous> (C:\Users\avina\Projects\Stackoverflow-Clone\client\node_modules\@typescript-eslint\typescript-estree\dist\parser.js:20:25)
    at Module._compile (C:\Users\avina\Projects\Stackoverflow-Clone\node_modules\v8-compile-cache\v8-compile-cache.js:192:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
```

Reply:
I see, then leave it as it is.